content/blog/2023/10/kubewarden-1.8-release.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Kubewarden 1.8.0 release
+authors:
+  - Víctor Cuadrado
+date: 2023-10-11
+---
+
+Today we are happy to announce the release of Kubewarden 1.8.0! 🎉🥳
+
+This is a small release, focused on [OpenTelemetry](https://opentelemetry.io/).
+
+The OpenTelemetry Protocol (OTLP) got its [first 1.0.0
+version](https://github.com/open-telemetry/opentelemetry-proto/releases/tag/v1.0.0)
+in July 2023, and the OpenTelemetry stack is becoming more stable; several
+libraries got their first 1.0.0 release, such as the [Go metric
+SDK](https://opentelemetry.io/blog/2023/otel-go-metrics-sdk-stable/) or the
+[.NET Automatic
+Instrumentation](https://opentelemetry.io/blog/2023/otel-dotnet-auto-instrumentation/).
+
+Still, the OTLP stack is not yet stable, and unannounced backwards-incompatible
+changes still happen. You can have a look at the status of each of their
+libraries and protocols [here](https://opentelemetry.io/status).
+
+For Kubewarden, this means we should expect more users to configure and fine
+tune the deployment to fit their stacks.
+
+## Changes
+
+With 1.8.0, the Kubewarden Controller now separately reconciles the configuration of both
+tracing and metrics for PolicyServers. In the past, enabling metrics in
+Kubewarden meant enabling both, and now users can decide to enable each separately, as we
+have realized from the requests raised in GitHub.
```

Comment:
maybe `in GitHub` can be rephrased with `from the community`

Reply:
Better! done in https://github.com/kubewarden/kubewarden.io/pull/195/commits/da56fe45efb8b26278c20dfd6626d670a90559d9.